src/Microsoft.DotNet.Darc/src/Darc/Helpers/LocalSettings.cs

```diff
@@ -107,7 +107,10 @@ public static DarcSettings GetDarcSettings(CommandLineOptions options, ILogger l
                         else if (parsedUri.Host == "dev.azure.com" || parsedUri.Host.EndsWith("visualstudio.com"))
                         {
                             darcSettings.GitType = GitRepoType.AzureDevOps;
-                            darcSettings.GitRepoPersonalAccessToken = localSettings != null ? localSettings.AzureDevOpsToken : options.AzureDevOpsPat;
+                            darcSettings.GitRepoPersonalAccessToken = 
+                                localSettings != null ? 
```

Comment:
Can you add some parentheses or indentation to be explicit about what this is doing? I can't really tell what its trying to do.